app/views/editor/choose.html.erb

```diff
@@ -0,0 +1,114 @@
+<style>
+  body, body>.container, body>.container>.row {
+    margin: 0;
+    padding: 0;
+    width: 100vw;
+    max-width: 100%;  }
+  body>.container {
+    padding-top: 0px;
+  }
+  .footer {
+    margin: 0;
+  }
+  #choose-screen { 
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    height: 65vh;
+    background:rgba(255,255,255,1);
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+    align-items: center;
+  }
+  .alert {
+    z-index: 600;
+  }
+  #choose-screen h2 {
+    font-size: 1.8em;
+    font-family: 'Junction Light';
+    margin-bottom: 1em;
+  }
+  #choose-screen .center-float {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 5vw;
+  }
+  #choose-screen .choose-box {
+    width: 240px;
+    height: auto;
+    margin: 10px;
+    padding: 10px;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.5em;
+  }
+
+  @media (max-width: 800px) {
+    #choose-screen {
+      height: 70vh;
+    }
+    #choose-screen .center-float {
+      flex-flow: column nowrap;
+    }
+    #choose-screen .choose-box {
+      width: 300px;
+      height: auto;
+      font-size: 1.2em;
+    }
+  }
+  @media (max-width: 300px) {
+    #choose-screen {
+    }
+    #choose-screen h2 {
+      font-size: 1.4em;
+    }
+    #choose-screen .choose-box {
+      width: 95vw;
+      font-size: 1.2em;
+    }
+  }
+</style>
+
+<div id="choose-screen">
```

Comment:
What if we used this but allow overriding it with a Feature? https://github.com/publiclab/plots2/blob/master/doc/DATA_MODEL.md#features

```ruby
<% if feature('post-chooser') == '' %>
  <%= feature('footer-notice') %>
<% else %>
  <!-- this content -->
<% end %>
```

What do you think?

Reply:
Sounds good! So basically the entire choose.html.erb file, including css?

Reply:
Yes, i think so -- that way, the admins on the site can make changes without a database change. This is nice because content types may be quite use-specific, and may change over time, but the site structure doesn't have to. Thanks!!!

Reply:
Makes sense!